glossary.ftd

```diff
@@ -1,22 +1,219 @@
 -- ft.page: Glossary
 
-For readers/writers of document by id, we have three kinds of id: `id: /foo/`,
-`full-id: /foo/-/x/`, `filename: foo.ftd`.
+An FPM Glossary
 
 
-Since we need to get `id`, `full-id` and `filename` of current document, to power
-document-readers, lets create processors `document-id`, `document-full-id` and
-`document-filename` as helpers as well. These three processors will return ids
-for current document.
+
+-- ft.h2: client project
+
+An [fpm project](glossary/#fpm-project) which is a clone to the
```

Comment:
This link is wrong. Relative links are with respect to current URL, and current URL is `/glossary/` already, so this link will resolve to `/glossary/glossary/#fpm-project`. Ideally we should use the glossary/term feature we discussed yesterday, but since it is not done yet, for same page link using just the fragment part, `[fpm project](#fpm-project)`.